hash/gbcolor.xml

```diff
@@ -434,7 +434,7 @@ license:CC0
 
 	<software name="aliceu" cloneof="alice">
 		<!-- Notes: GBC only -->
-		<!--    Retail cartridge of European version not yet secured    -->
+		<!-- Retail cartridge of European version not yet secured -->
```

Comment:
I think they intentionally indented the second line relative to the first as it’s also a note.  Same for a lot of the other comments in this file – subsequent notes are indented relative to the line beginning with “Notes:”.

Reply:
Almost every Notes was just a stand-alone GBC only tag which I've converted to proper xml tags in case somebody wants to actually use this metadata. This also nicely removes the "did they mean to indent this" ambiguity of the handful of comments.